pt/lessons/basics/collections.md

```diff
@@ -172,3 +172,19 @@ iex> map = %{foo: "bar", hello: "world"}
 iex> %{map | foo: "baz"}
 %{foo: "baz", hello: "world"}
 ```
+
+**Nota**: esta sintaxe funciona apenas para atualizar uma chave que já exista no mapa! Se a chave não existir, um `KeyError` será gerado.
```

Comment:
I think "... uma chave que já existe no mapa! ..." would translate better.

Reply:
You're right, @anderkonzen. I made the corrections. Thanks: :+1: